draft-ietf-quic-manageability.md

```diff
@@ -945,37 +945,63 @@ severe packet loss.
-the mitigation environment.  Limited first-packet garbage detection as in
-{{sec-garbage}} and stateful tracking of QUIC traffic as in {{sec-stateful}}
-above may be useful during classification.
+Efficient classification of this DDoS traffic in the mitigation environment
+is key to the success of this approach. Limited first-packet garbage detection
+as in {{sec-garbage}} and stateful tracking of QUIC traffic as in
+{{sec-stateful}} above may be useful during classification.
 
 Note that the use of a connection ID to support connection migration renders
 5-tuple based filtering insufficient and requires more state to be maintained by
-DDoS defense systems. For the common case of NAT rebinding, DDoS defense systems
+DDoS defense systems. For the common case of NAT rebinding, where the client's
+address changes without the client's intent or knowledge, DDoS defense systems
 can detect a change in the client's endpoint address by linking flows based on
-the server's connection IDs. QUIC's linkability resistance ensures that
+the server's connection IDs. However, QUIC's linkability resistance ensures that
 a deliberate connection migration is accompanied by a change in the connection
-ID.
-
-It is questionable whether connection migrations must be supported during a DDoS
-attack. If the connection migration is not visible to the network that performs
-the DDoS detection, an active, migrated QUIC connection may be blocked by such a
-system under attack. As soon as the connection blocking is detected by the
-client, the client may rely on the fast resumption mechanism provided by
-QUIC. When clients migrate to a new path, they should be prepared for the
-migration to fail and attempt to reconnect quickly.
-
-TCP syncookies {{?RFC4937}} are a well-established method of mitigating some
+ID. In this case, the connection ID can not be used to distinguish valid, active
+traffic from new attack traffic.
+
+It is also possible for
+endpoints to directly support security functions such as DoS
+classification and mitigation.
+Endpoints can cooperate with an in-network device directly by e.g.
+sharing information about connection IDs.
+
+Another potential method could use an
+on-path network device that relies on pattern inferences in the traffic and
+heuristics or machine learning instead of processing observed header
+information.
+
+However, it is questionable whether connection migrations must be supported
+during a DDoS attack. However, for unintentional mitigations, as is usually the
+case for NAT rebindings, the connection ID can be used to track
+connections (see {{sec-flow-association}}). If the connection
+migration is not visible to the network
+that performs the DDoS detection, an active, migrated QUIC connection may be
+blocked by such a system under attack. As soon as the connection blocking is
+detected by the client, the client may rely on the fast resumption mechanism
```

Comment:
Resumption might be fast, but still costs a lot more to handle than keeping a connection alive.  This won't produce a good outcome in the case of DoS.  I don't think that this is good advice at all.